rootconf/default/etc/init.d/gen-config.sh

```diff
@@ -17,9 +18,45 @@ gen_live_config()
 
 [ -r /lib/onie/gen-config-platform ] && . /lib/onie/gen-config-platform
 
+gen_machine_config()
+{
+    local build_conf=/etc/machine-build.conf
+    local live_conf=/etc/machine-live.conf
+    local machine_conf=/etc/machine.conf
+
+    gen_live_config > $live_conf
+
+    (cat <<EOF
```

Comment:
I don't like the subshell approach when using <<EOF.  I think this syntax is cleaner and easier to understand:
```
cat<<EOF > $machine_conf
# /etc/machine.conf for onie
# ...
EOF
```
You can also append by using `cat<<EOF >> $machine_conf`.

Please use that syntax here and below.